test/src/Server/ODM/CRUD/CRUDTest.php

```diff
@@ -1,490 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace LaminasTest\ApiTools\Doctrine\Server\ODM\CRUD;
-
-use DateTime;
-use Doctrine\Instantiator\InstantiatorInterface;
-use Doctrine\ODM\MongoDB\DocumentManager;
-use Laminas\ApiTools\ApiProblem\ApiProblem;
-use Laminas\ApiTools\ApiProblem\ApiProblemResponse;
-use Laminas\ApiTools\Doctrine\Admin\Model\DoctrineRestServiceEntity;
-use Laminas\ApiTools\Doctrine\Admin\Model\DoctrineRestServiceResource;
-use Laminas\ApiTools\Doctrine\DoctrineResource;
-use Laminas\ApiTools\Doctrine\Server\Event\DoctrineResourceEvent;
-use Laminas\Http\Request;
-use Laminas\ServiceManager\ServiceManager;
-use LaminasTest\ApiTools\Doctrine\TestCase;
-use LaminasTestApiToolsDbMongo\Document\Meta;
-use LaminasTestApiToolsGeneral\Listener\EventCatcher;
-use MongoClient;
-use PHPUnit_Framework_MockObject_MockObject;
-
-use function json_decode;
-use function json_encode;
-
-class CRUDTest extends TestCase
```

Comment:
The fact that we're starting to drop tests in order to keep things alive is a strong indicator that we should declare the BC break explicitly and move on here :thinking:

Reply:
We should, as dropping ODM support is a BC break.

Reply:
Agreed, but it's really for the best: ODM users are a drop in the ocean, and the complexity deriving from supporting both is mind-boggling.

To this day, I've yet to see a single software system where MongoDB is an architectural requirement for operation, so dropping support is a stance that I'm fully backing, and can defend long-term.